cmd/certsuite/run/run.go

```diff
@@ -68,8 +68,8 @@ func initTestParamsFromFlags(cmd *cobra.Command) error {
 	testParams.IncludeWebFilesInOutputFolder, _ = cmd.Flags().GetBool("include-web-files")
 	testParams.EnableDataCollection, _ = cmd.Flags().GetBool("enable-data-collection")
 	testParams.EnableXMLCreation, _ = cmd.Flags().GetBool("create-xml-junit-file")
-	testParams.TnfImageRepo, _ = cmd.Flags().GetString("tnf-image-repository")
-	testParams.TnfDebugImage, _ = cmd.Flags().GetString("tnf-debug-image")
+	testParams.CertsuiteImageRepo, _ = cmd.Flags().GetString("certsuite-image-repository")
+	testParams.CertsuiteDebugImage, _ = cmd.Flags().GetString("certsuite-debug-image")
```

Comment:
According to my previous comment, `certsuite-debug-image` --> `certsuite-probe-image-repository`.